config.sh

```diff
@@ -6,7 +6,7 @@
 function pre_build {
     # Any stuff that you need to do before you start building the wheels
     # Runs in the root directory of this repository.
-    if [ \( -n "$IS_OSX" -a "$MB_PYTHON_VERSION" = "2.7" \) -o \( "$MB_PYTHON_VERSION" = "3.6" \) ]; then
+    if [ "$MB_PYTHON_VERSION" = "3.6" ]; then
```

Comment:
it appears that `MB_PYTHON_VERSION` is passed on to the docker container as `PYTHON_VERSION` (without the `MB_` prefix). However, on the macOS multibuild environment, we don't have a `PYTHON_VERSION` variable, only an `MB_PYTHON_VERSION`, so to cover both you need to ... check both?
https://github.com/matthew-brett/multibuild/blob/548ebc80443250ead493164cba330e56b3558c70/travis_linux_steps.sh#L86

or just use `env_vars.sh`

sorry... :(

Reply:
you can see that it didn't catch the `MB_PYTHON_VERSION` (so tx isn't installed), otherwise something about pip installing tx would appear before this line:
https://travis-ci.org/adobe-type-tools/psautohint/jobs/424737479#L526

Reply:
oh, I'm so sorry Khaled I've misled you... So, the problem is that, it *is* now installing the afdko in the build environment (inside the CentOS docker) where the wheel is getting built. However, you want it inside the *test* environment, where you run the tests!
The latter is also a docker image (a vanilla Ubuntu Trusty). To configure the packages that gets installed in the test environment you have to set the `TEST_DEPENDS` environment variable.

We already use that in our .travis.yml, to install from dev-requirements.txt. To fix this you could add the `afdko` to dev-requirements.txt.

You could use an environment marker to do it conditionally only when `afdko; python_version == '3.6'` but writing that in dev-requirements.txt may be misleading..

Or we could make a tox environment where we install afdko and use that TOXENV...